pootle/templates/editor/units/xhr_checks.html

```diff
@@ -1,8 +1,9 @@
 {% load i18n locale staticpages %}
-{% if unit.get_qualitychecks.count %}
-<span class="sidetitle" lang="{{ LANGUAGE_CODE }}" title="{% trans 'Possible issues with the translation' %}">{% trans "Failing checks:" %}</span>
-<ul class="checks">
-  {% for check in unit.get_qualitychecks.iterator %}
+{% with unit.get_qualitychecks_critical as checks_critical %}
+{% if checks_critical.count %}
```

Comment:
I'd say there's no need for `.count`, as you're accessing the actual qs results right away. Also, if you'd only want to check for existence, `.exists` is your friend.